src/common.py

```diff
@@ -1,17 +1,35 @@
 keys_to_extract = [
-    "feedback_record_id",
     "created",
     "feedback",
     "url",
     "page_title",
-    "department",
+    "urgency",
+    "feedback_type",
     "primary_department",
     "document_type",
-    "feedback_type",
-    "urgency",
-    "publishing_app",
-    "locale",
-    "sentiment",
-    "spam_classification",
-    "labels",
+    # "department",
+    # "publishing_app",
+    # "locale",
+    # "sentiment",
+    # "spam_classification",
```

Comment:
Worth having a discussion about spam and primary dept vs dept here maybe

Reply:
Added to change log